packages/vuetify/src/components/VCalendar/modes/index.ts

```diff
@@ -0,0 +1,8 @@
+import { CalendarEventOverlapMode } from 'types'
+import { stack } from './stack'
+import { column } from './column'
+
+export const CalendarEventOverlapModes: Record<string, CalendarEventOverlapMode> = {
+  stack,
+  column,
+}
```

Comment:
Are there more than these two? if not, type can be strenghtened

Reply:
Not sure what you mean here. At the moment there are two, but others may be added.